backend/core-services/egov-workflow-v2/src/main/resources/egov-workflow-v2-persister.yml

```
@@ -273,4 +273,28 @@ serviceMaps:
       - jsonPath: $.BusinessServices.*.states.*.actions.*.auditDetails.lastModifiedBy
 
-      - jsonPath: $.BusinessServices.*.states.*.actions.*.auditDetails.lastModifiedTime
\ No newline at end of file
+      - jsonPath: $.BusinessServices.*.states.*.actions.*.auditDetails.lastModifiedTime
+
+  - version: 1.0
+    description: Persists pause state in the table (insert or update)
+    fromTopic: save-wf-pausestate
+    isTransaction: true
+    queryMaps:
+      - query: INSERT INTO eg_wf_pause_state (businessId, businessService, isPaused, comments, createdBy, lastModifiedBy, createdTime, lastModifiedTime) VALUES (?, ?, ?, ?::jsonb, ?, ?, ?, ?) ON CONFLICT (businessId, businessService) DO UPDATE SET isPaused = EXCLUDED.isPaused, comments = EXCLUDED.comments, lastModifiedBy = EXCLUDED.lastModifiedBy, lastModifiedTime = EXCLUDED.lastModifiedTime;
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)